package.json

```diff
@@ -15,7 +15,7 @@
     "react-confetti": "^6.1.0",
     "react-dom": "^17.0.2",
     "react-router-dom": "^6.3.0",
-    "react-scripts": "5.0.0",
+    "react-scripts": "^5.0.1",
```

Comment:
In general, you will want to be super careful about committing changes that update the package.json and package lock file. Reason being is because these changes might accidentally cause issues elsewhere.

In this case, it looks like you made a lot of changes to the package lock file so I am going to ask you to revert the changes made to the lock file and the package.json file back to its original state. 

The first answer here in this stackoverflow post should show you how to revert the changes for both the lock file and package.json file
https://stackoverflow.com/questions/59652396/change-package-lock-json-package-back-to-original-repo-version

In the future, make sure to always do a self review first of your changes to make sure you are committing only the changes necessary because if you make changes to the package.json file in larger projects they will probably ask you revert those changes or want to know the reasoning behind the change 👍